packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -208,9 +212,9 @@ const _useOrganizedParticipantsWithFocusedParticipants = (
   // get focused participants from map of remote participants in the order of the user ids
   const focusedParticipants: VideoGalleryRemoteParticipant[] = [];
   focusedParticipantUserIds.forEach((id) => {
-    const pinnedParticipant = remoteParticipantMap[id];
+    const pinnedParticipant = remoteParticipantMap[id as keyof VideoGalleryRemoteParticipant];
```

Comment:
is this `as` needed? Technically `id` isn't a `keyof VideoGalleryRemoteParticipant`, it can only be a string?

Reply:
Ah it shouldn't be needed any longer after you have the `{} as sortedRemoteParticipants` casted earlier on now